smartva/rules/homicide_child.py

```diff
@@ -8,8 +8,10 @@
 def logic_rule(row):
     value_of = value_from_row(row, int_or_float)
 
-    injury_by_other = value_of(Child.INFLICTED_BY_OTHER) == YES
-
+    injury_by_other = any([
+        value_of(Child.INFLICTED_BY_OTHER) == YES,
+        value_of(Child.VIOLENCE) == YES,
```

Comment:
This is going to result in substantially more violence deaths in the PHMRC validation data; it does make sense to add, but I don't like the rule-based approach.